cmd/internal/exec.go

```diff
@@ -23,8 +23,9 @@ import (
 	"strings"
 )
 
+// RunCmd prints the provided message and command and then executes it binding stdout and stderr
 func RunCmd(msg, cmd string, args ...string) error {
-	c := exec.Command(cmd, args...) // #nolint:gosec
+	c := exec.Command(cmd, args...) // nolint:gosec
```

Comment:
nit: I believe `nolint`-s should be without a space after `//`:
```suggestion
	c := exec.Command(cmd, args...) //nolint:gosec
```

See: https://github.com/golangci/golangci-lint#nolint

> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

Reply:
Changed project-wide